views/js/runner/plugins/controls/connectivity/connectivity.js

```diff
@@ -119,7 +119,7 @@ define([
 
                         //creates the waiting modal dialog
                         dialog = waitingDialog({
-                            message : __('You are encountering a prolonged connectivity loss. ') + message,
+                            message : __('You are encountering a prolonged connectivity loss. '),
```

Comment:
We could keep the message here but only include it if it is a string (see below).

Reply:
Then, we could:
L198: remove the passing of the error message: `self.displayWaitingDialog()`
L177: keep passing the (could be informational) message: no change required

Reply:
In this way, the message would be gone only in the case of connectivity error. And the change to current behaviour is minimised.

Reply:
Good point. Fixing it.